website/src/types/Organism.ts

```diff
@@ -28,6 +28,13 @@ export const organismConfig = {
         backgroundColor: 'bg-sandMuted',
         backgroundColorFocus: 'group-hover:bg-sand',
         borderEntryDecoration: 'hover:border-sand',
+        genome: [
+            {
+                gff3Source:
+                    'https://raw.githubusercontent.com/nextstrain/nextclade_data/refs/heads/master/data/nextstrain/sars-cov-2/wuhan-hu-1/orfs/genome_annotation.gff3',
```

Comment:
Did you discuss whether it makes sense to include those files into this repo and download them from the Astro server rather than from GitHub?

Do we expect frequent changes in these files? If yes, we should keep the link to GitHub. Otherwise I think it would make sense that we host them ourselves (similar to as we do for the map data already).